### ISSUE TITLE
DEPR: pd.merge vs DataFrame.merge

### ISSUE THREAD

**Issue by jbrockmendel** [pandas-dev/pandas#53708]
### Pandas version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://pandas.pydata.org/docs/whatsnew/index.html) of pandas.

- [X] I have confirmed this bug exists on the [main branch](https://pandas.pydata.org/docs/dev/getting_started/install.html#installing-the-development-version-of-pandas) of pandas.

### Reproducible Example

```python
NA
```

### Issue Description

should we deprecate one to get to Just One Way to do it?

### Expected Behavior

NA

### Installed Versions

<details>

Replace this line with the output of pd.show_versions()

</details>

**Comment by tpaxman** [comment]
If one were deprecated, I would vote to keep the `DataFrame` method, as I suspect it is used most often and because it lends itself to method chaining, e.g., `df1.merge(df2, on='a').merge(df3, on='b')` whereas the module function would be less readable (e.g., `pd.merge(pd.merge(df1, df2, on='a'), df3, on='b')`).

**Comment by samukweku** [comment]
Anything wrong with having both? Is it a maintenance issue? Pd.merge is nice sometimes for me when merging named series

**Comment by rsm-23** [comment]
I agree with @tpaxman , `df1.merge(df2, on='a').merge(df3, on='b')` is much more intuitive for chaining and readability.

**Comment by jbrockmendel** [comment]
Closing as not worth the churn.
